styles.css

```diff
@@ -0,0 +1,391 @@
+*{
+	box-sizing: border-box;
+}
+
+body {
+    max-width: 80rem;
+    margin: auto;
+}
+
+
+header {
+	height:10%;
+	background-color:#093C60;
+	color:white;
```

Comment:
Try not to mix and match how you're referencing colors. If you're using white, you can use #FFFFFF